build.sbt

```diff
@@ -8,3 +8,5 @@ addSbtPlugin("com.github.gseitz" % "sbt-release" % sbtReleaseVersion)
 addSbtPlugin("com.jsuereth" % "sbt-pgp" % sbtPgpVersion)
```

Comment:
I don't think this is needed for `addSbtPlugin()`, which automatically inserts cross build version info in.